cluster/storage/src/test/java/com/linkedin/openhouse/cluster/storage/selector/impl/StorageInNameRegexSelectorTest.java

```diff
@@ -0,0 +1,129 @@
+package com.linkedin.openhouse.cluster.storage.selector.impl;
+
```

Comment:
can we add a e2e test which picks up certain cluster.yaml and we create 2 tables in there with different storages (file: vs hdfs)? and we can validate based on the get response?

Reply:
StorageSelectorConfigTest.testProvideStorageInNameRegexSelector() already tests the different storage scenario. I will also perform the different storage scenario in docker and update the testing done section.

Reply:
TY!  Unit tests for the added modules are good! thanks for that. I meant an e2e unit test with inmemory H2 backend for example:  com.linkedin.openhouse.tables.e2e.h2.TablesControllerTest. 
This will make reproducing errors/bugs easier and increase stability of the selector code.

Reply:
Can we add the e2e test as part of implementing the FeatureToggleStorageSelector which we intend to use in production for migrations? 

I've replicated the e2e test with multiple storages in docker-compose and added to the testing section.